apps/roam/src/components/settings/HomePersonalSettings.tsx

```diff
@@ -114,6 +116,37 @@ const HomePersonalSettings = ({ onloadArgs }: { onloadArgs: OnloadArgs }) => {
           </>
         }
       />
+
```

Comment:
This is a test button that can be used to test the feature, once its tested we can make it a stub